packages/clerk-js/rspack.config.js

```diff
@@ -369,24 +385,32 @@ const prodConfig = ({ mode, analysis }) => {
 
   // webpack-bundle-analyzer only supports a single build, use clerkBrowser as that's the default build we serve
   if (analysis) {
-    return clerkBrowser;
+    return [clerkBrowser];
   }
 
   return [clerkBrowser, clerkHeadless, clerkHeadlessBrowser, clerkEsm, clerkCjs];
 };
 
+/**
+ *
+ * @param {object} config
+ * @param {'development'|'production'} config.mode
+ * @param {object} config.env
+ * @returns
+ */
 const devConfig = ({ mode, env }) => {
   const variant = env.variant || variants.clerkBrowser;
   // accept an optional devOrigin environment option to change the origin of the dev server.
   // By default we use https://js.lclclerk.com which is what our local dev proxy looks for.
   const devUrl = new URL(env.devOrigin || 'https://js.lclclerk.com');
 
+  /** @type {() => import('@rspack/core').Configuration} */
   const commonForDev = () => {
     return {
       module: {
         rules: [svgLoader(), ...typescriptLoaderDev(), clerkUICSSLoader()],
       },
-      plugins: [new ReactRefreshPlugin({ overlay: { sockHost: devUrl.host } })],
+      plugins: [new ReactRefreshPlugin(/** @type {any} **/ ({ overlay: { sockHost: devUrl.host } }))],
```

Comment:
This cast is to workaround a type issue that I've opened a PR for. https://github.com/rspack-contrib/rspack-plugin-react-refresh/pull/9